src/nft/components/bag/Bag.tsx

```diff
@@ -320,17 +352,17 @@ const Bag = () => {
               Continue
             </Box>
           )}
-        </Column>
+        </BagContainer>
       ) : (
-        <Column zIndex={isMobile || isOpen ? 'modalOverTooltip' : '3'} className={styles.bagContainer}>
```

Comment:
If the modal is hidden when closed, do we need the lower zIndex of 3?

Reply:
yeah it lowers to 3 when the modal is hidden and closed - its in the styled component now

Reply:
I'm wondering if this needs to be set since the component is hidden

Reply:
oh it won't be hidden the isModal is referring to the black screen that goes below it